_data/nav.yml

```diff
@@ -1,3 +1,7 @@
+- title: Caffe2 is being deprecated - Read more
```

Comment:
Can we say "Caffe2 APIs are being deprecated - Read more"

Reply:
Updated. Thanks!